templates/examples/olinkclient/main.cpp.tpl

```diff
@@ -3,12 +3,15 @@
 #include "apigear/olink/olinkclient.h"
 #include "olink/clientregistry.h"
 #include <memory>
-
+{{- $features := .Features }}

```

Comment:
You can always access the root context with `$` so `$.Features` should always work. No need to change.

Reply:
thx